targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8C63XX/TARGET_MCU_PSOC6_M4/hex/README.md

```diff
@@ -3,7 +3,7 @@ README for pre-compiled PSoC 6 Cortex M0+ core images
 
 This folder contains precompiled program images for the CM0+ core of the PSoC 6(63xx) MCU suitable for use with MBed OS applications running on CM4 core. Two images are available:
 
-* `psoc63_m0_default_1.01.hex`
+* `psoc63_m0_default_1.02.hex`
```

Comment:
Does this follow your versioning (not just dropping latest image here as a replacement rather than remove + add ) ?

Reply:
Yes, we'd like to keep control of the binary version being used, so we can be sure all the fixes are in place.